yarn-project/foundation/src/json-rpc/server/json_rpc_server.ts

```diff
@@ -25,7 +25,7 @@ export class JsonRpcServer {
     private objectClassMap: JsonClassConverterInput,
     /** List of methods to disallow from calling remotely */
     public readonly disallowedMethods: string[] = [],
-    private log = createDebugLogger('aztec:foundation:json-rpc:server'),
+    private log = createDebugLogger('json-rpc:server'),
```

Comment:
Move all `json-rpc` loggers to their own namespace (previously only the client used it, while the serve was part of foundation).